models/attribute-definition/src/constants.ts

```diff
@@ -0,0 +1,10 @@
+const attributeConstraints = [
```

Comment:
My issue with using the array for a constant value rather than an object map is that we can't later reference a specific item in a preset, for example `attributeConstraints.SameForAll`

Reply:
Ahh! I would've missed that. 🙏  
For the generator, what is the preferred way to grab only the object map values? `Object.values`?

Reply:
Yessir!

Reply:
WHEEEEE